### PR TITLE
Ou/test/settings quota display check

### PR DESCRIPTION
**Pull request by OliwierUrbanski**
🧪 Quota Display Check (Settings > User)
This PR introduces a complete test for validating the quota usage display inside the HiDrive Next app under Settings → User.

✅ What’s tested:
The correct storage percentage is shown and is between 0–100%, allowing ±1% rounding tolerance.

The usage string follows the expected format:
xx MB of XX GB in use

The used MB value is verified to always be greater than 0.

Output is formatted nicely: 0.04 GB of 1 000 GB used (0 %)

A UI dump (ui_dump.xml) is written to disk for debugging any XPath changes.

### CODE REVIEW

File: TestplanHDNX/test_settings_quota_display_check.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+
+# --------------------------------------------------------------------------- #
+# Appium-Utilities
+# --------------------------------------------------------------------------- #
+def start_appium_session() -> webdriver.Remote:
+    opts = Capabilities.get_options()
+    driver = webdriver.Remote("http://localhost:4723", options=opts)
+    driver.implicitly_wait(8)
+    return driver
```
Comment (bogus78) [inline TestplanHDNX/test_settings_quota_display_check.py]: do we need this initialization and preparation in all test classes? move them to a helper class to avoid duplicate code and use reusable code for better maintainability

File: TestplanHDNX/test_settings_quota_display_check.py
Comment (bogus78) [inline TestplanHDNX/test_settings_quota_display_check.py:1]: for what is this commit?

**Comment by bogus78** [comment]
add also a env.example with needed empty variables and a note to the readme

File: TestplanHDNX/test_settings_quota_display_check.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+# --------------------------------------------------------------------------- #
+# GUI-Vorbereitung – pyautogui
+# --------------------------------------------------------------------------- #
+def prepare_gui() -> None:
+    """HiDrive Next in den Vordergrund holen und »Einstellungen« öffnen."""
+    pyautogui.click(3073, 12);  time.sleep(0.3)   # Menüleisten-Icon
+    pyautogui.click(2855, 86);  time.sleep(0.3)   # Benutzer-Dropdown
+    pyautogui.click(2858, 289); time.sleep(0.3)   # «Einstellungen»
```
Comment (bogus78) [inline TestplanHDNX/test_settings_quota_display_check.py]: The coordinates can also be used as constants

**Comment by bogus78** [comment]
proper logging - use a logger instead of print for better monitoring and debugging

File: TestplanHDNX/test_settings_quota_display_check.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    opts = Capabilities.get_options()
+    driver = webdriver.Remote("http://localhost:4723", options=opts)
+    driver.implicitly_wait(8)
+    return driver
+
+
+def click_when_ready(driver, xpath: str, desc: str = "", timeout: int = 20):
+    elem = WebDriverWait(driver, timeout).until(
+        EC.element_to_be_clickable((By.XPATH, xpath))
+    )
```
Comment (bogus78) [inline TestplanHDNX/test_settings_quota_display_check.py]: extract this WebDriverWait into a helper class for reusable code

**Comment by bogus78** [comment]
I would suggest to build a kind of a framework pattern, contains all related methods like click(), wait(), findElement() ... and gui related classes. a test class should be as readable as possible. But this can be done in a separate branch.

**Review by bogus78 — COMMENTED** [review]
please review your commits, some of them do have useful changes and some of them can be squashed / removed.

please move the GUÍCooddinates into a global class as well. I would like to have just on place to change the coordinates:

```
class GuiCoordinates:
    MENU_ICON = (3073, 12)
    USER_DROPDOWN = (2860, 86)
    SETTINGS_ITEM = (2865, 289)
    CLICK_PAUSE = 0.3

```
`WebDriverWait().until() `methods can also be reused, as it can also be included in a global class

**Comment by OliwierUrbanski** [comment]
review feedback applied - ready for review

**Review by bogus78 — APPROVED** [review]
review ok, but please apply the helper classes for the other tests as well

**Comment by OliwierUrbanski** [comment]
review feedback applied - ready for review @bogus78

**Review by bogus78 — APPROVED** [review]
